keps/sig-network/2104-reworking-kube-proxy-architecture/README.md

```diff
@@ -574,6 +594,165 @@ func (epc *EndpointsClient) Dial() (conn *grpc.ClientConn, err error) {
 
 ... JAY/RAJAS/MIKAEL (TODO, add a good "client.Run" or "PreRun" example here ... 
 
```

Comment:
@astoycos can you add a section on "how to implement a new backend" in this KEP? You wanted to do that anyways and @danwinship just mentinoed a diagram would help 

https://jayunit100.blogspot.com/2022/08/how-kpng-api-objects-connect.html heres a diagram im working on but... separating the processes will help also

Reply:
Working on this...